CONTRIBUTING.md

```diff
@@ -0,0 +1,205 @@
+   need to update all PRs with the `devnet-ready` label to instead have the `on-devnet` label
+   (TODO: automate this upon merge). The administrator will then deploy `devnet`.
+7. Once the `on-devnet` label appears on your PR, if you are a core team member it is your
+   responsibility to verify that the features/changes/fixes introduced by your PR are
+   functioning properly on `devnet` by interacting with the live network. If you are an
+   external contributor, a core team member will be assigned to test this for you.
+8. If your feature/change/fix is confirmed working on `devnet`, the `devnet-pass` label should
+   be added. Otherwise if there are issues, the `devnet-fail` label should be added and you
+   will need to make changes to your PR and repeat the previous steps in this process. In some
+   cases a revert PR will need to be created reverting your changes from the `pre-devnet` and
+   `devnet` branches, respectively.
+9. Once `devnet-pass` has been added to your PR, it is eligible for inclusion in the next
+   `testnet` deploy. We typically run `testnet` deploys every other wednesday.
+10. On the appropriate date, an administrator will open a PR merging the current `devnet`
+    branch into `testnet`. This PR should include a bulleted list of all PRs included in the
+    deploy so they can be easily found after the fact (TODO: automate this). The PR should
+    exclude any PRs that currently have the `devnet-fail` label via a revert (TODO: enforce via
+    CI). This PR will require two reviews from the core team as a sanity check. After merging
+    into `testnet`, the administrator will then need to run the deploy and update all PRs
+    included in the deploy with the `on-testnet` label (TODO: automate this upon merge). Next
+    the administrator must cut a (pre-release) release in GitHub for `testnet` (TODO: github
+    action to generate the release and release notes).
+11. Once the `on-testnet` label appears on your PR, if you are a core team member it is your
+    responsibility to once again verify that the features/changes/fixes introduced by your PR
+    are functioning properly on `testnet` by interacting with the live network, if applicable.
+    If you are an external contributor, a core team member may be assigned to do this testing
+    for you but otherwise it will be your responsibility to show evidence on the PR that the
+    testing is successful. Once this has been verified, the `testnet-pass` label should be
+    added. If testing fails, the `testnet-fail` label should be added and PRs should be opened
+    reverting the change from `devnet-ready`, and then a PR should be opened merging the
+    modified `devnet` into `testnet`. These revert PRs, if they occur, _must_ be merged before
+    a new deploy can be run (TODO: enforce this via CI).
+12. After the SOP period (72 hours on testnet) has passed and the `testnet-pass` label has been
```

Comment:
is this still valid? given the new release cycle (wednesday - wednesday), I think period lasts at least a week